tests/behaviour/steps/inspectcli.py

```diff
@@ -171,3 +219,341 @@ def step_impl(context, error_message: str):
     except Exception as ex:
         assert ex is not None
         assert ex.message == f"{error_message} {context.csvw_metadata_file}"
+
+
+@Then("the Type printable is validated for single-measure pivoted data set")
+def step_impl(context):
+    result_type_info: CSVWType = context.result_type_info
+    assert result_type_info is not None
+    assert result_type_info == CSVWType.QbDataSet
+
+
+@Then("the Catalog Metadata printable is validated for single-measure pivoted data set")
+def step_impl(context):
+    result_catalog_metadata: CatalogMetadataResult = context.result_catalog_metadata
+    assert result_catalog_metadata is not None
+    assert result_catalog_metadata.title == "Pivoted Shape Cube"
+    assert result_catalog_metadata.label == "Pivoted Shape Cube"
+    assert result_catalog_metadata.issued == "2022-10-28T13:35:50.699296"
+    assert result_catalog_metadata.modified == "2022-10-28T13:35:50.699296"
+    assert result_catalog_metadata.license == "None"
+    assert result_catalog_metadata.creator == "None"
+    assert result_catalog_metadata.publisher == "None"
+    assert (
+        len(result_catalog_metadata.landing_pages) == 1
+        and result_catalog_metadata.landing_pages[0] == ""
+    )
+    assert (
+        len(result_catalog_metadata.themes) == 1
+        and result_catalog_metadata.themes[0] == ""
+    )
+    assert (
+        len(result_catalog_metadata.keywords) == 1
+        and result_catalog_metadata.keywords[0] == ""
+    )
+    assert result_catalog_metadata.contact_point == "None"
+    assert result_catalog_metadata.identifier == "qb-id-10004"
+    assert result_catalog_metadata.comment == "None"
+    assert result_catalog_metadata.description == "None"
+
+
+@Then(
+    "the Data Structure Definition printable is validated for single-measure pivoted data set"
+)
+def step_impl(context):
+    result_qube_components: QubeComponentsResult = context.result_qube_components
+    assert result_qube_components is not None
+
+    components = result_qube_components.qube_components
+    assert len(components) == 5
+
+    component = get_dsd_component_by_property_url(
+        components, "qb-id-10004.csv#dimension/some-dimension"
+    )
+    assert_dsd_component_equal(
```

Comment:
These tests are so specific I'm not sure they really fit into the behaviour testing framework. 

Could you convert them into unit tests? Unless we absolutely need any of the docker-based functionality.

Reply:
We already have unit tests for this. If we are moving away from the test based approach for behave tests, we should test the objects? DSD components are objects.